apps/web/components/events/EventCardPreview.tsx

```diff
@@ -0,0 +1,42 @@
+import Image from 'next/image';
+import { Event } from '@our-scene/api-hooks/resources/events/types';
+
+type EventCardProps = {
+  event: Event;
+  handleSelect: (event: Event) => void | Promise<void>;
+  isFocused: boolean;
+};
+
+export const EventCardPreview = ({ event, handleSelect, isFocused }: EventCardProps) => {
+  const { title, blurb, address, start, end } = event;
+
+  const startDate = new Date(start);
+  const endDate = new Date(end);
+  const formattedStartDate = startDate.toLocaleString();
+  const formattedEndDate = endDate.toLocaleString();
+
+  const focusedClasses = 'shadow-xl card card-compact card-side bg-slate-100';
+  const nonFocusedClasses = 'shadow-xl card card-compact card-side bg-base-100';
```

Comment:
[nit-pick] because only the last class in each of these is different, we can clean this up a little
this it purely a nit to keep it a little more DRY
```js
let cardClasses = 'shadow-xl card card-compact card-side'
cardClasses = cardClasses + isFocussed ? 'bg-slate-100' : 'bg-base-100'
```

Reply:
I totally agree with this, the suggestion was helpful for how to reformat was helpful!

I just pushed these changes up, so now on line 23 you should see
``className={`${cardClasses} ${isFocused ? 'bg-slate-100' : 'bg-base-100'}` }``